DEVELOPMENT.md

```diff
@@ -29,6 +29,9 @@ This repo currently contains two examples:
 | **Model (Pipelines)** | `tensorflow-housing`     | `bike-rentals-auto-ml` |
 | **App (ACM)**         | `tensorflow-housing-app` | `bike-rental-app`      |
 
+
+*Additionally*, you can follow the [OpenShift AI tutorial - Fraud detection example](https://access.redhat.com/documentation/en-us/red_hat_openshift_ai_self-managed/2.8/html/openshift_ai_tutorial_-_fraud_detection_example/index) to execute the end to end workflow of training a model from scratch & building the [inferencing container](pipelines/containerfiles/Containerfile.openvino) to deliver to Edge environments.
```

Comment:
If we are going to promote the fraud detection tutorial (which I like), shouldn't we just expand all the relevant places and PipelineRuns to actually lead the reader through the steps, and also have things tested by the CI?

Reply:
This is a maintenance liability.

Is it critical that this goes in in this PR? If not, I'd just leave this change out.

Reply:
> Is it critical that this goes in in this PR? If not, I'd just leave this change out.

Which change, this text in the DEVELOPERS doc?  I don't have any particular need to included this the document other than to point people to documented workflow that is supported and fits our E2E scenario training and delivering models to the edge.

 > This is a maintenance liability.

I am curious about this. Is it a maintenance liability because it is based on code outside of this repository?

Reply:
I get that us writing here that a user can follow that tutorial and use our pipeline for build might mean that it stops working in the future when we don't test it permanently, however, we can at least say it, that it is not permanently tested by us and it is best effort from our side.

Reply:
These are all good points.  I am going to separate all of the "Fraud Detection Model" specific changes, this DEVELOPERS.md and Containerfile, into a separate story/work since some of the longterm fixes should be aligned with a complete rewrite of the pipeline, tasks and manifests.

> I get that us writing here that a user can follow that tutorial and use our pipeline for build might mean that it stops working in the future when we don't test it permanently,

IMO, this scenario does have a benefit of us dogfooding our own tutorials since this tutorial and official documentation  are maintained by another group working on OpenShift AI.  If we find a bug, it would be good to file it early in the upstream releases.

Reply:
Just pointing people somewhere with some promise of experiencing the end-to-end example across two repos, without including the relevant steps in our documentation, will lead to people (including myself) to get stuck with "OK, what next?" questions, either because the steps are not trivial or because the other repo has changed significantly so our assumptions in the Containerfile no longer work.

Folks will either abandon the attempt, or they will file issues here (better) or in the https://github.com/rh-aiservices-bu/fraud-detection (worse, maintainers of that repo don't know anything about this work). even issues filed against this repo will be hard to respond to in a timely and effective manner if we don't have the experience with that other repo among the team and in the CI.